### PR TITLE
state catchup APIs

### PR DESCRIPTION
**Pull request by jbearer**
Implements endpoints to fetch fee and block state from the most recent decided view and later undecided views.

Only thing that's missing is an end-to-end test for non-empty fee accounts, because we don't yet have a good way of initializing the fee ledger with some pre-funded accounts. However, we have a separate unit test to make sure the membership proofs work, and we have an end-to-end test for absence proofs. So I think we can merge this as-is, and I will add the end-to-end membership proof test later after #1048

### CODE REVIEW

File: sequencer/api/state.toml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+[route.blocks]
+PATH = ["/catchup/:view/blocks", "/blocks"]
+":view" = "Integer"
+DOC = """
+Get the blocks Merkle tree frontier.
+
+This endpoint can be used to catch up to the current state from a recent state by fetching the
+frontier at the given `:view` number. It can also be used to query the latest finalized state.
+
+Returns the blocks Merkle tree frontier -- the path to the most recently appended leaf, relative to
```
[the remark this answers is not in the document]
Comment (jbearer) [reply sequencer/api/state.toml:34]: Right now the JSON is just the serialization of some Jellyfish type, so I'm not sure it's especially useful to document the exact structure at this point
